tethys_apps/migrations/0006_auto_20240401_2032.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 3.2.23 on 2024-04-01 20:32
+
+from django.db import migrations, models
+
+
+def forward(apps, schema_editor):
+    """From proxyapp with logo_url field to icon field"""
+    ProxyApp = apps.get_model("tethys_apps", "proxyapp")
+
+    db_alias = schema_editor.connection.alias
+
+    for cs in ProxyApp.objects.using(db_alias).all():
+        cs.icon = cs.logo_url
+        cs.save()
+
+
+def backward(apps, schema_editor):
+    """From proxyapp with icon field to logo_url field"""
+    ProxyApp = apps.get_model("tethys_apps", "proxyapp")
+
+    db_alias = schema_editor.connection.alias
+
+    for cs in ProxyApp.objects.using(db_alias).all():
+        cs.logo_url = cs.icon
```

Comment:
This could fail if a user migrates, adds static files as icons and then un-migrates. I'm not too concerned about that scenario though. I'm not sure what would happen (if it would drop the data or refuse to undo the migration).

Reply:
oh good point. I can do some testing with this. What would be the desired logic? Raise an exception and let the user know that the logo_url needs to be a url? Raise a warning and make the field empty or make it a fake http url with the static file address?

Reply:
@ckrew I'm not too concerned about this, so I think we should just move forward as is. @swainn thoughts?

Reply:
I think the only ones who would unmigrate, would be us during development. It would be very unlikely for our users, so I'm not worried about it either.